lib/module/src/module.rs

```diff
@@ -7,7 +7,7 @@
 
 use Backend;
 use cretonne_codegen::entity::{EntityRef, PrimaryMap};
-use cretonne_codegen::result::{CtonError, CtonResult};
+use cretonne_codegen::result::CtonError;
```

Comment:
It occurs to me that we may want to rename `CtonError` to `CodegenError` or now, though we don't necessarily need to do that in this PR.